examples/mxnet_simple.py

```diff
@@ -0,0 +1,131 @@
+"""
```

Comment:
Please add this example to the example list in [examples/README.md](https://github.com/pfnet/optuna/blob/master/examples/README.md#examples-with-ml-libraries).

Reply:
In addition, please add `mxnet` as a package that installed when the daily CI is executed (see: [.circleci/config.yml](https://github.com/pfnet/optuna/blob/master/.circleci/config.yml#L191)).

Reply:
Okay.

Reply:
Done.